ngraph/frontend/paddlepaddle/src/op/softplus.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <node_context.hpp>
+
+#include "default_opset.hpp"
+
+namespace ngraph {
```

Comment:
Please use ov namespace

Reply:
hi @ilyachur  do we have to change namespace from ngraph to openvino for all paddle frontend ops? or we simply change it for new developed code files?

If it is required for all files, I am going to do it in a separate PR.